src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
```

Comment:
Instead of calling `ColorSupplier.getRandomColor()` call it on the instance of `ColorSupplier` class. 
You can add it as member of the current class (`Lottery`) and initialize via constructor.